serializations.md

```diff
@@ -438,6 +438,7 @@ purging them from the tree.
 , <lock_period>      :: int()
 , <ttl>              :: int()
 , <fee>              :: int()
+, <delegates>        :: [id()]
```

Comment:
The serialization of the channel object itself must be [updated](https://github.com/aeternity/epoch/blob/4f3b0f7cdc841dda3a45668138f39f498ce54ef0/apps/aechannel/src/aesc_channels.erl#L199) as well.